lib/ansible/config/base.yml

```diff
@@ -36,15 +36,27 @@ ANSIBLE_COW_SELECTION:
   env: [{name: ANSIBLE_COW_SELECTION}]
   ini:
   - {key: cow_selection, section: defaults}
-ANSIBLE_COW_WHITELIST:
-  name: Cowsay filter whitelist
+ANSIBLE_COW_ENABLED:
+  name: Cowsay filter acceptance list
   default: ['bud-frogs', 'bunny', 'cheese', 'daemon', 'default', 'dragon', 'elephant-in-snake', 'elephant', 'eyes', 'hellokitty', 'kitty', 'luke-koala', 'meow', 'milk', 'moofasa', 'moose', 'ren', 'sheep', 'small', 'stegosaurus', 'stimpy', 'supermilker', 'three-eyes', 'turkey', 'turtle', 'tux', 'udder', 'vader-koala', 'vader', 'www']
   description: White list of cowsay templates that are 'safe' to use, set to empty list if you want to enable all installed templates.
-  env: [{name: ANSIBLE_COW_WHITELIST}]
+  env:
+  - name: ANSIBLE_COW_WHITELIST
+    deprecated:
+      why: Normalizing names to new standard.
+      version: "2.14"
+      alternatives: 'ANSIBLE_COW_ENABLED'
+  - name: ANSIBLE_COW_ENABLED
```

Comment:
Shouldn't this get a `version_added: '2.11'`? And shouldn't the deprecation version above be 2.15 (for a standard four version deprecation cycle)?

Reply:
yep, this was started at a time i thought it would still make 2.10